netlify.toml

```diff
@@ -0,0 +1,21 @@
+# example netlify.toml
```

Comment:
- [ ] It would be better if we identify some part of our code we found interesting or we are proud of in our video presentation